Package.swift

```diff
@@ -1,4 +1,4 @@
-// swift-tools-version:5.3
+// swift-tools-version:5.5
 
```

Comment:
All these changes seem fine to me. Out of curiosity, does the comment at the top of this file actually stop the build process with an informative error for builders with out of date compilers, or will those old builders fail with some obscure error when we try to use unsupported features?

Reply:
The "comment" is functional and determines which language mode and version of the package library is used. Trying to use an older version of Swift will give an error message saying that the tools version isn't supported.

Reply:
Glad to hear that!